README.md

```diff
@@ -268,16 +268,27 @@ $ mbed remove text-lcd
 
 After importing a program or creating a new one, you need to tell mbed CLI where to find the toolchains that you want to use for compiling your source tree.
 
-There are two ways to do this:
-* Through the mbed CLI configuration.
-* Via mbed_settings.py file in the root of your program, which is automatically created (if it doesn't already exist). 
+There are multiple ways to do this:
+* `mbed_settings.py` file in the root of your program, which is automatically created if it doesn't already exist. 
+* The mbed CLI configuration
+* setting an environment variable
```

Comment:
Probably make the 's' of "setting" capital here, to make it consistent with the others.